site/docs/components/tag/examples.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+<LivePreviewControls>
+<LivePreview componentName="tag" exampleName="Primary">
+
+## Primary
+
+Primary tags feature subtle background colors by default, making them ideal for presenting information or content without being overly attention-grabbing.
+
+To add an optional border, simply pass the `bordered` prop to the component.
+
+### Best practices
+
+- Ensure tag colors are consistent with their associated categories.
```

Comment:
This is weird because from a user's perspective categories = color. What I guess this is saying is use categories to represent things consistently

Reply:
hahaha i know what both mean, still sounds confusing... 
Maintain consistent color-coding for each category.?